.env

```diff
@@ -4,3 +4,4 @@ ETHCONNECT_TOPIC=token
 ETHCONNECT_PREFIX=fly
 FACTORY_CONTRACT_ADDRESS=
 AUTO_INIT=true
+PASSTHROUGH_HEADERS=X-SuperCoolHeader,X-RayVision
```

Comment:
I would be careful about actually setting these here - I think this file will actually be picked up as defaults if you don't overwrite the vars in your Docker config or elsewhere.

Reply:
Thanks. Didn't actually mean to check this one in